webapp/crashstats/crashstats/tests/test_views.py

```diff
@@ -201,7 +202,7 @@ def test_html(self):
         handler500 = getattr(views, end)
 
         # to make a mock call to the django view functions you need a request
-        fake_request = RequestFactory().request(**{"wsgi.input": None})
+        fake_request = RequestFactory().request(**{"wsgi.input": StringIO("")})
```

Comment:
Turns out when you pass `None` in as a value, something in Django internals calls `.read()` on it and then everything gets mad. `wsgi.input` should be a stream, so I replaced it with a `StringIO` and that seemed to work.